demo/lego_official_projects/TRACK3R.py

```diff
@@ -35,7 +35,7 @@ def __init__(self, medium_motor=OUTPUT_A, left_motor=OUTPUT_B, right_motor=OUTPU
 
     def fire_ball(self, state):
         if state:
-            self.medium_motor.run_to_rel_pos(duty_cycle_sp=100, position_sp=3*360)
+            self.medium_motor.run_to_rel_pos(speed_sp=100, position_sp=3*360)
```

Comment:
`duty_cycle_sp` and `speed_sp` use different units of measurement, so you are also changing the effective speed here. For large motors you can multiply by 10 to get close enough. e.g. This should be `speed_sp=100`.